TodoListClient/app/components/forms/todo-item-form/component.js

```diff
@@ -1,9 +1,44 @@
-import Component from '@ember/component';
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Changeset from 'ember-changeset';
+import lookupValidator from 'ember-changeset-validations';
+import TodoItemValidations from '../../../validations/todo-item';
 
-export default Component.extend({
-  actions: {
-    save() {
-      this.saveAction();
+export default class TodoItemForm extends Component {
+  @service notify;
+  @service router;
+
+  constructor() {
+    super(...arguments);
+
+    // Was unable to use the changeset helpr, so I'm constructing the changeset manually.
+    // Most likely related to this issue, but the fix didn't work.
+    // https://github.com/poteto/ember-changeset-validations/issues/214
```

Comment:
I don't really like constructing the variable manually either, so I'll look into this/open an issue. I'm hoping I can play around with it and get it to work as expected.

I also removed Chrome. I downloaded since some unit tests in another module I made a PR for needed it and I think my browser just remembered my last download path where I put the logo.